docs/02-Test with Evergreen/05-Use the API/01-REST-V2-Usage.md

```diff
@@ -221,6 +221,8 @@ Task Annotations give users more context about task failures.
 | note             | note_object            | Comment about the task failure                                                                                   |
 | issues           | []issue_link           | Links to tickets definitely related                                                                              |
 | suspected_issues | []issue_link           | Links to tickets possibly related                                                                                |
+| metadata_links   | []metadata_link        | List of arbitrary links associated with a task, meant to be displayed in the task metadata sidebar               |
```

Comment:
small note, should we mention that right now this is limited to one?

Reply:
Sure, I'll add that